packages/shared/src/styles/base.css

```diff
@@ -337,6 +337,8 @@ html.light .invert .invert,
   position: relative;
   flex-direction: column;
   align-items: stretch;
+  min-height: -webkit-fill-available;
```

Comment:
This might have some unwanted side effects i rather test.
For now can't we fix the button position with zindex?